drivers/python/rethinkdb/net.py

```diff
@@ -515,6 +523,12 @@ def __init__(self, conn_type, host, port, db, auth_key, timeout, ssl, **kwargs):
         self._instance = None
         self._next_token = 0
 
+    def client_port(self):
+        return self._instance._socket.client_port()
```

Comment:
I think it would be slightly better to have this be a properly. We can get a read-only property by just putting @property on the line before the `def`. `client_address` should probably get this as well. I am ambivalent about those on `SocketWrapper` since that is not deliberately exposed.

Additionally I am slightly concerned about what happens if `self._instance` or `self._instance._socket` are `None` when this is called (say when a `Connection` is `closed()`).